hw/ip/otbn/dv/tracer/README.md

```diff
@@ -0,0 +1,141 @@
+# OTBN Tracer
```

Comment:
Just realised this is a very RTL tracer focused README, we might want it more as a generic here's the trace format and here's how the RTL tracer provides you with trace records. Just requires a bit of rejigging of headings.